hw12/Lesson12.py

```diff
@@ -0,0 +1,119 @@
+import json
+from pprint import pprint
+
+
+def write_(data, filename):
+    data = json.dumps(data)
+    data = json.loads(str(data))
```

Comment:
what is the reason why did you use `dumps` and `loads` and then again `dump`?

Reply:
I Agree, these 2 rows are unnecessary there and have been deleted.